lib/mix_test_interactive/settings.ex

```diff
@@ -181,5 +194,10 @@ defmodule MixTestInteractive.Settings do
     end
   end
 
+  defp args_from_settings(%__MODULE__{tags: [_ | _] = tags}) do
+    tag_args = Enum.map(tags, &["--only", &1]) |> List.flatten()
+    {:ok, tag_args}
+  end
+
```

Comment:
I suggest making the pattern match look more like the one for the `patterns` case. Also, `Enum.map |> List.flatten` can be simplified to `Enum.flat_map`.

#optional

```suggestion
  defp args_from_settings(%__MODULE__{tags: tags}) when length(tags) > 0 do
    tag_args = Enum.flat_map(tags, &["--only", &1])
    {:ok, tag_args}
  end
```